tests/Unit/OutputFormatterTest.php

```diff
@@ -307,18 +307,18 @@ public function provideUnchangedStringForRemoveLastSemicolon(): array
     {
         return [
             'empty string' => [''],
-            'string without semicolon' => ['Tea or coffee?'],
-            'string with trailing semicolon' => ['Tea or coffee;'],
-            'string with semicolon in the middle' => ['Tea; coffee'],
-            'string with semicolons in the middle and trailing' => ['Tea; coffee;'],
+            'string without semicolon' => ['Earl Grey: hot'],
```

Comment:
I'd prefer these to be lowercase and hyphenated, so that they look like CSS properties.

```suggestion
            'string without semicolon' => ['earl-grey: hot'],
```